cardano-launcher/src/Cardano/Shell/Launcher.hs

```diff
@@ -27,8 +29,11 @@ module Cardano.Shell.Launcher
 import           Cardano.Prelude hiding (onException)
 
 import           Prelude (Show (..))
+import           Data.Aeson (FromJSON, ToJSON(toJSON), genericParseJSON, genericToJSON, defaultOptions)
+import           Data.Yaml as Y
 import qualified System.Process as Process
 import           Turtle (system)
+import           Test.QuickCheck (Arbitrary (arbitrary), elements)
```

Comment:
Not sure why this is here, are you testing this?
Also, it would be simpler to write `Arbitrary (..)`.